.vaunt/config.yaml

```diff
@@ -0,0 +1,77 @@
+version: 0.0.1
+start_date: "2023-12-02" #change this to the date you want to start tracking from
```

Comment:
Vaunt does not support variables in the yaml like this. So may want to remove this and put the date instead in each spot this is used.

Reply:
Hello @jeff1010322,

I wanted to inquire if it's feasible to introduce variable support in Vaunt. We're in the process of expanding our rewards program with three additional tiers. Implementing a centralized variable system would significantly streamline our workflow and enhance efficiency

Reply:
Hi, 
I think it is more so that the Yaml format does not support variables in the fields like this.
The closest thing you can do as far as I am aware is use alias and anchor to set reusable field, but it has to be the whole string field itself. So would look something like this:

```
variables:
  bug_tracker_condition: &bug_tracker_condition count() >= 1 & created_at >= 2023-12-02 #change this to the date you want to start tracking from

achievements:
  - achievement:
      name: Bug Trackers
      icon: "Badge Icon URL Placeholder"
      description: Awarded for creating issues in the project.
      triggers:
        - trigger:
            actor: author
            action: issue
            condition: *bug_tracker_condition
```